runelite-client/src/main/java/net/runelite/client/util/ColorUtil.java

```diff
@@ -131,6 +142,23 @@ public static String colorToAlphaHexCode(final Color color)
 		return String.format("%08x", color.getRGB());
 	}
 
+	/**
+	 * Gets the same RGB color with the specified alpha value.
+	 *
+	 * @param color The RGB color to use.
+	 * @param alpha The alpha value to use (0-255).
+	 * @return      A Color with the given RGB and alpha.
+	 */
+	public static Color colorOfAlpha(final Color color, int alpha)
```

Comment:
This should be named `withAlpha`. The color **of** an alpha value will always be gray.